.gitignore

```diff
@@ -21,3 +21,7 @@ node_modules/
 !.yarn/releases
 !.yarn/sdks
 !.yarn/versions
+
+### IDE
+*.iml
+*.idea
```

Comment:
unrelated changes. Should be in a different PR or you can set that in your user home folder.

Reply:
> unrelated changes. Should be in a different PR or you can set that in your user home folder.

Would it be fine if I make a new PR for this?

Reply:
I will make new PR.